src/cpuinfo_x86.c

```diff
@@ -1244,6 +1246,13 @@ static void DetectSseViaOs(X86Features* features) {
   features->ssse3 = GetDarwinSysCtlByName("hw.optional.supplementalsse3");
   features->sse4_1 = GetDarwinSysCtlByName("hw.optional.sse4_1");
   features->sse4_2 = GetDarwinSysCtlByName("hw.optional.sse4_2");
+#elif defined(CPU_FEATURES_OS_FREEBSD)
+  features->sse = sysctlbyname("hw.instruction_sse", NULL, NULL, NULL, 1);
```

Comment:
As far as I understand, this is not how `sysctlbyname` is supposed to be used.
https://www.freebsd.org/cgi/man.cgi?query=sysctlbyname&manpath=FreeBSD+13.0-RELEASE+and+Ports

One need to set proper arguments.
Also returned value is `0` upon success and `-1` upon failures.
I believe we should rename `GetDarwinSysCtlByName` above and use it here.